inc/html.class.php

```diff
@@ -3920,9 +3920,9 @@ static function initEditorSystem($name, $rand = '', $display = true, $readonly =
       });";
 
       if ($display) {
-         echo  Html::scriptBlock($js);
+         echo  Html::scriptEnd($js);
```

Comment:
`scriptEnd` takes no arguments.

I'm not sure what is the best between your proposal, and keep `scriptBlock` and remove `scriptStart`. @orthagh ?

Reply:
I agree, the question is real, but we should choose one or the other, but of course not a mix :)
It seems that I've been a bit too quick :)

Reply:
Huh and by the way... With current proposal, `$js` is never sent to the buffer; it should not works

Reply:
Looking at the code; `scriptBlock` is used about 69 times, where `scriptStart|End` is used only twice.